README.dist.md

```diff
@@ -1,46 +1,71 @@
-`ddev wp core download`
+Run `ddev start` to start the site. This will install WordPress files, composer packages, npm packages, and start the DDEV server. Once the server is started DDEV will automatically start Vite for local development. 
+
+If this is the first time this project has been set up: 
+1. Select the desired language.
+2. Fill out the site information.
+3. Then click "Install WordPress"
+4. Once WordPress has been set up login with your user information.
 
-Start local developement server
-`bin/start`
+If this is an existing project, get an database download from production, staging or another team member. Steps to download a database are different depending on the hosting server. 
+
+Once you have the database file import the database into DDEV using:
+```bash
+ddev import-db --file=FILENAME.sql.gz
+```
+Then login using the username/password for the user on that database.
 
-1. Open a browser and navigate to [local site](https://wpstarter.ddev.site).
-2. Select English as the language.
-3. Fill out the site information.
-4. Then click "Install WordPress"
-5. Once WordPress has been set up login with your user information.
+You are all ready to start working on the site.
 
-If you need to build production files. Change the DDEV config.yaml `ENVIRONMENT`
-to `prod` and then run `bin/build`. Which will build the JS and CSS files in the dist folder and out put a manifest file.
+### Build for production
+The deploy script should build the files for production, but if you want to test that out on your local server you can change the DDEV config.yaml `ENVIRONMENT` to `prod` and then run `ddev npm run build`. This will build the JS and CSS files in the dist folder and out put a manifest file.
 
-## Blocks
+### Theme.json
+The `theme.json` holds a lot of the core WordPress theme settings. The `theme.json` is build using several js files in `/src/theme-json`, Vite builds all of these files and exports a `theme.json` for both `dev` and `build`. Do not edit directly `theme.json` as it will be over written on build. 
+
+Several of the Tailwind variables are pulled in and Tailwind should be used as the primary way to style elements. If you need to, you can pull in more Tailwind variable for custom styling in `theme.json`.
+
+## Plugins
+* [Advanced Custom Fields PRO](https://www.advancedcustomfields.com/pro/)
+* *List other Plugins used*
+
+## Custom Blocks
 Builds are build using ACF and core WordPress blocks. Styles for the blocks are in `src/styles/blocks`.
 
-- Accordion
-- Alert Banner
-- CTA
-- Image Caption
-- Logo Grid
-- Text Icon Cards
-- Text Image
-- Video Embed
+* Accordion
+* Alert Banner
+* CTA
+* Image Caption
+* Logo Grid
+* Text Icon Cards
+* Text Image
+* Video Embed
+* *List other custom Blocks*
 
 ## Commands
 
-```
-ddev npm run dev #builds local files
-ddev npm run build  #builds production files
+```bash
+ddev start
+ddev rebuild
+ddev stop
+ddev npm run dev #builds local
```

Comment:
We might want to add a note below this section letting devs know that it *shouldn't* be necessary to run this command since it's automatically ran in the background of `ddev start`